src/test/countryDetails.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import Countries from '../components/country';
+import store from '../redux/store';
+
+describe('Test for countryDetails', () => {
```

Comment:
**Test Passed** 👍

![2](https://user-images.githubusercontent.com/63972670/225973812-16f62a2e-e32f-4111-9949-0d516a9c675e.PNG)

Reply:
Thank you